5-contribute/label-data.md

```diff
@@ -7,8 +7,8 @@ but you can also contribute data from your own community.
 
 ### Why contribute labeled data
 
-Submitting labeled data lets the Perspective API team train the models on more
-comments, and makes it possible for the API to improve and learn from your
+Submitting labeled data lets the Perspective API team train the attributes on
```

Comment:
Same thought as comment above--we train models, not attributes, so this might be a good place to say "model attributes" to be clearer,